collection-scripts/gather

```diff
@@ -28,7 +28,7 @@ check_managed_clusters() {
 }
 
 check_if_hub () {
-    HUBNAMESPACE=`oc get multiclusterhubs.operator.open-cluster-management.io --all-namespaces --no-headers=true| awk '{ print $1 }'`
+    HUBNAMESPACE=`oc get multiclusterengines.multicluster.openshift.io --all-namespaces --no-headers=true| awk '{ print $1 }'`
```

Comment:
Since the purpose of this call is to get the namespace of the multiclusterhub, switching it to get the multiclusterengine will not work because it does not have a namespace